### PR TITLE
bug_fix_partitioned_path

### PR DESCRIPTION
**Pull request by DanielElisenberg**
Include version in partitioned parquet file path.

Should write some unit tests for partitioned data as well.

### CODE REVIEW

**Comment by DanielElisenberg** [comment]
Added a unit test in test_processor.py testing one happy case event query for an dataset with temporality type: accumulated

File: data_service/adapters/storage/local.py [hunk 1]
```diff
@@ -23,11 +23,9 @@ def __create_download_path(self, dataset_name: str, version: str) -> str:
             f"{self.settings.DATASTORE_DIR}/data/"
             f"{dataset_name}/{dataset_name}__{version}"
         )
-        print(f"VERSIONSSS {path}")
```
Comment (vaskalan) [inline data_service/adapters/storage/local.py]: A small typo?

**Review by vaskalan — APPROVED** [review]

File: data_service/adapters/storage/local.py (hunk 1 above)
Comment (DanielElisenberg) [reply data_service/adapters/storage/local.py]: yes 😅 this is removed in the last commit

**Comment by pawbu** [comment]
👍

**Pull request merged**